torchdynamo/utils.py

```diff
@@ -219,6 +219,12 @@ def clone_input(x):
 
 
 def clone_inputs(example_inputs):
+    if isinstance(example_inputs, dict):
+        for key, value in example_inputs.items():
```

Comment:
```suggestion
        example_inputs = dict(example_inputs)
        for key, value in example_inputs.items():
```

So we don't mutate the caller's copy of `example_inputs`